### ISSUE TITLE
image field/tag missing

### ISSUE THREAD

**Issue by YunaBraska** [w3stling/rssreader#8]
Hey I would like to have the availability to also get unknown fields xml like like `image` (e.g. https://worldoftanks.eu/en/rss/news/)

thx

**Comment by w3stling** [comment]
Fixed in version 2.5.0
Added [getImage()](https://w3stling.github.io/rssreader/javadoc/2.5.0/com.apptastic.rssreader/com/apptastic/rssreader/Channel.html#getImage()) method to `Channel` class.

Example:
```java
RssReader rss = new RssReader();
Optional<Image> image = rss.read("https://worldoftanks.eu/en/rss/news/")
                           .findFirst()
                           .map(Item::getChannel)
                           .flatMap(Channel::getImage);
```

**Comment by YunaBraska** [comment]
thx
